hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/CustomGasCharging.java

```diff
@@ -96,6 +95,8 @@ public void maybeRefundGiven(
      * Tries to charge gas for the given transaction based on the pre-fetched sender and relayer accounts,
      * within the given context and world updater.
      *
+     * <p>Even if there are gas charges, still returns the intrinsic gas cost of the transaction.
```

Comment:
Is this comment accurate?  It implies to me that we will always return intrinsic gas

Reply:
Ugh typo, thanks! Fixing on different branch.